docs/using-keycloak.md

```diff
@@ -49,7 +49,7 @@ docker run -d --restart=always \
     -e KEYCLOAK_USER=admin \
     -e "KEYCLOAK_PASSWORD=<admin_password_here>" \
     -e DB_VENDOR="MYSQL" \
-    jboss/keycloak
+    jboss/keycloak:3.4.3.Final
```

Comment:
`jboss/keycloak:3.4` is probably also good. We could leave the minor version open and assume mysql and keycloak don't make breaking changes in minor versions.

Reply:
hm...keycloak:3.4 does not exist. We will have to go with a minor version.